resources/js/views/settings/roomTypes/Index.vue

```diff
@@ -233,8 +241,8 @@ export default {
 
     tableFields () {
       const fields = [
-        { key: 'description', label: this.$t('settings.roomTypes.description'), sortable: true },
-        { key: 'short', label: this.$t('settings.roomTypes.icon'), sortable: true }
+        { key: 'description', label: this.$t('settings.roomTypes.description'), sortable: true, tdClass: 'td-max-width-0-lg' },
+        { key: 'short', label: this.$t('settings.roomTypes.icon'), sortable: true, tdClass: 'td-max-width-0-lg', thStyle: { width: '10%' } }
```

Comment:
why do you use the `td-max-width-0-lg` when the text truncate is not used in the cell?